server/pension/models.py

```diff
@@ -34,30 +115,28 @@ def __str__(self):
         return '{} / {}'.format(self.quarter, self.year)
 
 
-class ManagingBody(models.Model):
-    """
-    The body that owns the funds.
-    """
+class Fund(models.Model):
```

Comment:
@ranmoshe I don't have a problem with changing `ManagingBody` to `Fund`. But in the `views` and `serializer` you expose the `issuer` when before we expose the `ManagingBody` (that now is `Fund`). So, why don't expose the `Fund` now?

As I understand the main page (you can see it in the [link](https://drive.google.com/drive/u/0/folders/0B7RkqHKoREPqRnZiRnZoWFF1ek0)) need the `Fund` and not the `Issuer`.

Also, you changed it now, why don't write a `doc`? (: